### PR TITLE
fix: change loading skeleton gradient for dark mode

### PR DESCRIPTION
**Pull request by adam-strzelec**
## Description

Change gradient color for loading skeleton in dark mode

## Testing

- Set app to dark mode
- Open any loading skeleton while fetching data

**Changes** 🏗

Before:

Screenshot 2024-10-17 at 15 07 28

After:

Screenshot 2024-10-17 at 15 08 09

Resolves [#31415](https://github.com/JoinColony/colonyCDapp/issues/3293)

### CODE REVIEW

**Review by rdig — APPROVED** [review]
This seems to be all good, at least as much as I've tested.

However I want to ask you to be more specific with your testing instructions.

> Open loading skeleton

This is very generic, and I can't really tell _which_ loading skeleton I need to test

Screenshot from 2024-10-22 16-19-38
Screenshot from 2024-10-22 16-19-57

**Review by iamsamgibbs — CHANGES_REQUESTED** [review]
I'm afraid this might be a much bigger bit of work than initially anticipated.

This fix only applies to places where the LoadingSkeleton component is used.

So it works great here:
Screenshot 2024-10-22 at 17 08 25

But the fix isn't applied anywhere that just uses the `skeleton` class and not the LoadingSkeleton, such as here:

Screenshot 2024-10-22 at 17 09 30

There are at least 47 places where it is used just as a class name and having to use the `usePageThemeContext` hook everywhere just to apply this change is going to be a real pain. Is there definitely no way to make this change in the tailwind config to use this background-image for the `skeleton` class when in dark mode so we can avoid adding the `skeleton-dark` class everywhere?

**Comment by arrenv** [comment]
47 file changes does not seem worth the effort at all for a slight color change, so, if this cannot be unified as a part of this PR then I would suggest scrapping this issue.

**Review by rdig — CHANGES_REQUESTED** [review]
In light of recent changes I will rescind my approval

**Comment by adam-strzelec** [comment]
@arrenv @rdig @mmioana @iamsamgibbs You're right. I didn't notice that skeleton can be called in other ways in the application than just through LoadingSkeleton. I've fixed that and it should work in the whole application now.

**Review by iamsamgibbs — APPROVED** [review]
Nice solution, this seems to work for me.

Screenshot 2024-10-24 at 08 34 56

Screenshot 2024-10-24 at 08 35 07

Screenshot 2024-10-24 at 08 35 13

Light mode is unaffected:

Screenshot 2024-10-24 at 08 35 57

Screenshot 2024-10-24 at 08 36 04

**Review by mmioana — APPROVED** [review]
Definitely a global and more clean approach to solving this issue 💯 

Nice work @adam-strzelec 🙌 

Dark mode
Screenshot 2024-10-24 at 10 26 15
Screenshot 2024-10-24 at 10 27 07

Light mode
Screenshot 2024-10-24 at 10 26 25
Screenshot 2024-10-24 at 10 26 50

File: tailwind.config.js [hunk 1]
```diff
@@ -250,8 +253,7 @@ module.exports = {
             'background-color': theme('colors.gray.100'),
             'background-repeat': 'no-repeat',
             'min-height': '1em',
-            'background-image':
-              'linear-gradient(90deg,rgba(255, 255, 255, 0),rgba(255, 255, 255, 0.5),rgba(255, 255, 255, 0))',
+            'background-image': `linear-gradient(90deg,rgba(255, 255, 255, 0),${theme('colors.gradient.base')},rgba(255, 255, 255, 0))`,
```
Comment (rumzledz) [inline tailwind.config.js:256]: Yessssss yes yes yes 💯

**Review by rumzledz — APPROVED** [review]
Clean and wonderful fix @adam-strzelec !! ✨ 

dark mode loader

**Comment by adam-strzelec** [comment]
@rdig I think I need your approval to merge it as now the merging is blocked.

**Review by rdig — APPROVED** [review]

**Comment by rdig** [comment]
@adam-strzelec done! :100:

**Pull request merged**